internal/service/certificatesigningrequest.go

```diff
@@ -165,6 +168,15 @@ func (h *ServiceHandler) CreateCertificateSigningRequest(ctx context.Context, cs
 	if errs := csr.Validate(); len(errs) > 0 {
 		return nil, api.StatusBadRequest(errors.Join(errs...).Error())
 	}
+	// Crypto validation
+	cn, ok := ctx.Value(middleware.TLSCommonNameContextKey).(string)
+	if ok {
```

Comment:
What do we do if `!ok`?

Reply:
Not OK means we are running with auth disabled. We simply skip all of this as it does not apply.